From 3092e86c8754ae524ad09210e39bc53180262d7b Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 26 Feb 2013 20:03:27 +0000
Subject: configure: correct path of run dir

We're defaulting to /var/gdm instead of /var/run/gdm now, woops.
---
diff --git a/configure.ac b/configure.ac
index 0918060..c2de8b3 100644
--- configure.ac
+++ configure.ac
@@ -1359,7 +1359,7 @@ AC_ARG_WITH([run-dir],
 if ! test -z "$with_run_dir"; then
    GDM_RUN_DIR=$with_run_dir
 else
-   GDM_RUN_DIR=${localstatedir}/gdm
+   GDM_RUN_DIR=${localstatedir}/run/gdm
 fi
 
 AC_SUBST([GDM_RUN_DIR])
--
cgit v0.9.1

